packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -89,6 +89,13 @@ export default class BuySellStore extends BaseStore {
 
     @computed
     get rendered_items() {
+        const filtered_items = this.items.filter(item => {
+            if (this.table_type === 'buy') {
+                return item.type === 'sell';
+            }
+            return item.type === 'buy';
+        });
```

Comment:
Suggestion:
- You can define `buy` and `sell` as constant variable so you don't have to update their values in different places.
- You can also simplify the condition
```suggestion
        const filtered_items = this.items.filter(item => {
             this.table_type === type.BUY ? item.type === type.SELL : item.type === type.BUY;
        });
```